ext/bcmath/libbcmath/src/num2str.c

```diff
@@ -55,18 +56,17 @@ zend_string *bc_num2str_ex(bc_num num, size_t scale)
 	if (signch) *sptr++ = '-';
 
 	/* Load the whole number. */
-	nptr = num->n_value;
```

Comment:
Maybe move the declaration of the ``const char`` to here, as the comment was confusing me without the allocation